platform/linuxbsd/display_server_x11.cpp

```diff
@@ -4435,10 +4435,19 @@ Vector<String> DisplayServerX11::get_rendering_drivers_func() {
 DisplayServer *DisplayServerX11::create_func(const String &p_rendering_driver, WindowMode p_mode, VSyncMode p_vsync_mode, uint32_t p_flags, const Vector2i &p_resolution, Error &r_error) {
 	DisplayServer *ds = memnew(DisplayServerX11(p_rendering_driver, p_mode, p_vsync_mode, p_flags, p_resolution, r_error));
 	if (r_error != OK) {
-		OS::get_singleton()->alert("Your video card driver does not support any of the supported Vulkan or OpenGL versions.\n"
-								   "Please update your drivers or if you have a very old or integrated GPU, upgrade it.\n"
-								   "If you have updated your graphics drivers recently, try rebooting.",
-				"Unable to initialize Video driver");
+		if (p_rendering_driver == "vulkan") {
+			OS::get_singleton()->alert("Your video card driver does not support the selected Vulkan version.\n"
+									   "Please try updating your GPU driver or try using the OpenGL 3 driver.\n"
+									   "You can enable the OpenGL 3 driver by starting the engine from the \n"
+									   "command line with the command: 'godot --rendering-driver opengl3'.\n"
```

Comment:
It's worth noting that *end users* also see this message, and in their case what they have is probably "CrueltySquad2.exe" and not a "godot" binary. But it's good enough for now I think, we can improve this further by adding a link to a dedicated troubleshooting page in the documentation eventually.

Reply:
I guess it can use `OS::get_singleton()->get_executable_path().get_file()`.
In case of macOS, it probably should be something like `vformat("open %s --args --rendering-driver opengl3", OS::get_singleton()->get_executable_path().get_base_dir().plus_file("../..").simplify_path())`.

Reply:
Updated to use bruvzg's suggestions! Thank you. I can't test on macOS, so I would appreciate confirmation that it doesn't break anything

Reply:
Do you know if there is anything we can do to get a wider alert dialogue on macOS? On Linux it appears to scale based on the width of the text

Reply:
It is possible to add `accessoryView` to the alert to force a bigger size, but default text is always the same width.

Reply:
https://github.com/godotengine/godot/pull/67564